Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -411,6 +411,25 @@ static void getImageBytesFromVideoFrame(WebGPU::Queue& backing, const RefPtr<Vid
 }
 #endif
 
+#if PLATFORM(COCOA) && ENABLE(VIDEO) && ENABLE(WEB_CODECS)
+static void clipTo8bpc(std::span<const uint8_t> data, size_t bitsPerComponent, Vector<uint8_t>& byteSpanBacking)
```

Comment:
Just a nit, but maybe it's worth avoiding the `bpc` abbreviation?

Reply:
good idea, expanded to `bitsPerChannel`